pkg/agent/openflow/client.go

```diff
@@ -536,10 +539,20 @@ func (c *client) InstallPodFlows(interfaceName string, podInterfaceIPs []net.IP,
 	isAntreaFlexibleIPAM := c.connectUplinkToBridge && c.nodeConfig.PodIPv4CIDR != nil && !c.nodeConfig.PodIPv4CIDR.Contains(podInterfaceIPv4)
 
 	localGatewayMAC := c.nodeConfig.GatewayConfig.MAC
-	flows := []binding.Flow{
-		c.featurePodConnectivity.podClassifierFlow(ofPort, isAntreaFlexibleIPAM),
-		c.featurePodConnectivity.l2ForwardCalcFlow(podInterfaceMAC, ofPort),
+
+	var flows []binding.Flow
+	if c.enableMulticluster {
+		// If multi-cluster is enabled, we store podClassifierFlow with a different key in
+		// cache, in order to reconcile this flow easier when the Pod update its labels.
+		podClassifierFlow := c.featurePodConnectivity.podClassifierFlow(ofPort, isAntreaFlexibleIPAM, UnknownLabelIdentity)
```

Comment:
could we just add a parameter labelIdentity to InstallPodFlows and make this function the only one managing the pod flows? UpdatePodClassifierFlowsWithNewLabelID is not needed in that case and code should be simpler. We don't expect labelID changes very often unless user changes the label of Pod or Namespace, right?

Reply:
> could we just add a parameter labelIdentity to InstallPodFlows and make this function the only one managing the pod flows? UpdatePodClassifierFlowsWithNewLabelID is not needed in that case and code should be simpler. We don't expect labelID changes very often unless user changes the label of Pod or Namespace, right?

My previous thought is `InstallPodFlows` will install all flows related to this Pod while init/add and `UpdatePodClassifierFlowsWithNewLabelID` only manage/update its classifier flow. 
If we only use this func to manage pod flows, then we may need another parameter to determine whether we need to install all flows or only update classifier flow. We can't use if the passed in `labelID` is `unknownID` or not to decide, since it's possible that we only need to update its classifier flow to `unknownID`. 
Hope I understood what you mean.

Reply:
> Is it easier if the key in the flow cache is always "interfaceName/classifier", no matter multipleCluster is enbaled or not? The difference is only the parameter for labelIdentity in these two cases. Then we can delete both the flows using keys of "interfaceName/classifier" and "interfaceName" in UninstallPodFlows ?

Yes, IMO we can always use another cache key for classifier flows. Before I'm just not sure if doing this will make the cache complicate/confused in single cluster.

Reply:
I meant it seems fine to just override all flows related to this Pod to be simpler. It should just work, or we could improve the underlying interface to only update flows that actually change.

Reply:
Got it. Changed.